### PR TITLE
vulkan-*: Update to 1.3.216.0, new port

### PR DESCRIPTION
**Pull request by judaew**
- glslang: Update to 11.10.0
- spirv-tools: Update to 2022.2
- spirv-headers: New port (ver. 1.3.216.0)
- vulkan-{headers,loader,tools,validationlayers}: Update to 1.3.216.0

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.4 21F79 x86_64
Command Line Tools 13.4.0.0.1.1651278267

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by Gcenx** [comment]
There’s already a PR open to update MoltenVK https://github.com/macports/macports-ports/pull/14858

**Comment by judaew** [comment]
Thank you @Gcenx, `MoltenVK` version 1.1.9 will be sufficient for the `vulkan-loader`, so I will drop this commit.

**Comment by Gcenx** [comment]
Got chance to look over this some more and the thing that stands out is the way you’ve handled python.

The usual “Macports way” would be to actually patch the python calls to point to the Macports python binary.

However if python is only being called during configure/build/destroot/test I simply use the following
```
set py_ver                  3.10
set py_ver_nodot            [string map {. {}} ${py_ver}]
foreach stage {configure build destroot test} {
    ${stage}.env-append     PATH=${frameworks_dir}/Python.framework/Versions/${py_ver}/bin:$env(PATH)
}

depends_build-append python${py_ver_nodot}

```

**Comment by judaew** [comment]
Thank you for this snippet, I will use it now.

**Comment by judaew** [comment]
Done

File: graphics/vulkan-loader/Portfile [hunk 1]
```diff
@@ -4,33 +4,36 @@ PortSystem          1.0
...
+homepage            https://vulkan.lunarg.com
 
-depends_build-append port:MoltenVK port:vulkan-headers port:python37
+depends_build-append \
+                    port:MoltenVK \
+                    port:python310 \
+                    port:vulkan-headers
 
 # MoltenVK is x86_64-only
 supported_archs     x86_64
```
Comment (Gcenx) [inline graphics/vulkan-loader/Portfile:28]: MoltenVK is now arm64 x86_64

Comment (judaew) [reply graphics/vulkan-loader/Portfile]: Thank you, done

**Comment by judaew** [comment]
@mascguy Does looks it good to you?

**Review by mascguy — APPROVED** [review]

**Pull request merged**